paella-matterhorn/plugins/edu.harvard.dce.infoButtonPlugin/dce_infobutton.js

```diff
@@ -14,18 +14,28 @@ Class ('paella.plugins.InfoPlugin', paella.ButtonPlugin,{
     var thisClass = this;
 
     var popUp = jQuery('<div id="dce-info-popup"></div>');
-    var buttonActions =[ 'Help with this player', 'Report a problem', 'Feedback', 'All Course Videos' ];
+    var buttonActions =[ 'Help with this player', 'Report a problem', 'DCE system status', 'Feedback', 'All Course Videos' ];
 
+    jQuery(popUp).append(thisClass.getItemTitle());
```

Comment:
Jay, I'm reusing the syntax "jQuery(popUp)" as used by UPV Paella, see following lines below.